attach/frida_uprobe_attach_impl/src/frida_attach_utils.cpp

```diff
@@ -47,8 +47,10 @@ resolve_function_addr_by_module_offset(const std::string_view &module_name,
 			get_module_base_addr(std::string(module_name).c_str());
 	}
 	if (!module_base_addr) {
-		SPDLOG_ERROR("Failed to find module base address for {}",
-			     module_name);
+		// It's not a bug, it might be attach to a unrelated process
```

Comment:
Warning will be better? Attaching to unrelated processes is not expected

Reply:
Since we are using LD_PRELOAD, there might be usecases where the user sets it as env variables to trace all commands in a bash, or the subprocesses of the program, e.g. `perf`



It's not an error or warning in these cases.